client/pages/auth-form.jsx

```diff
@@ -25,8 +26,9 @@ export default class Auth extends React.Component {
     }
   }
 
-  handleSubmit(event) {
+  handleSignUp(event) {
     event.preventDefault();
+    this.setState({ isSigningUp: false });
```

Comment:
You should use React Router to control which screen is showing. If you want to adopt a technology into your project, that's fine, but be sure to go "all in". Switching screens with React Router is controlled by `<Link>` and `navigate()`

https://reactrouter.com/docs/en/v6/api#link

https://reactrouter.com/docs/en/v6/api#usenavigate